README.md

```diff
@@ -120,9 +111,21 @@ $ make
 ```
 
 To test VDSM (refer to `tests/README` for further tests information):
-```shell
-$ make check
-```
+
+- Before running VDSM tests, activate the environment:
+  ```shell
+  $ source ~/.venv/vdsm/bin/activate
+  ```
+
+- To run the entire test suit:
+  ```shell
+  $ make check
+  ```
+
+- When done, you can deactivate the environment:
+  ```shell
+  $ deactivate
+  ```
```

Comment:
Moving the instructions how to activate and deactivate the virtual environment to the section about running the tests makes sense, but it conflicts with Albert changes. Let's wait until this work is done.